src/core/data.jl

```diff
@@ -250,24 +250,28 @@ function _apply_func!(data::Dict{String,<:Any}, key::String, func)
     end
 end
 
-
 "Transforms network data into per-unit"
 function make_per_unit!(data::Dict{String,<:Any})
+    _IM.modify_data_with_function!(data, _pm_it_name, _make_per_unit!; apply_to_nws = false)
+end
```

Comment:
A better adaptation of this type of function would be to move the check `if !haskey(data, "per_unit") || data["per_unit"] == false ...` into this function and then use `_IM.modify_data_with_function!(...; apply_to_nws = true)` to apply `_make_per_unit_nw!` to all networks.

Reply:
Ah, yes, this makes sense. Didn't think of that.